namadadryrun/chain.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "namadahousefire",
```

Comment:
I haven't checked the PR fully, but I think `chain_name` should be `namadadryrun` here and in the assetlist.json.

Reply:
ah ok thanks! I completely missed that

Reply:
fixed!